### PR TITLE
Fix docs for UdpSocket::send 

### LINKED ISSUE #652: docs for UdpSocket::send have same example as send_to
cf. https://docs.rs/async-std/1.4.0/async_std/net/struct.UdpSocket.html#method.send

### PR DESCRIPTION
**Pull request by Noah-Kennedy**
Minor change to alter documentation for `send` to no longer duplicate the documentation for `send_to`.
Fixes #652.

### CODE REVIEW

File: src/net/udp/mod.rs [hunk 1]
```diff
@@ -297,12 +300,11 @@ impl UdpSocket {
     /// #
     /// use async_std::net::UdpSocket;
     ///
-    /// let socket = UdpSocket::bind("127.0.0.1:0").await?;
+    /// let socket = UdpSocket::bind("127.0.0.1:7878").await?;
     /// socket.connect("127.0.0.1:8080").await?;
+    /// let bytes = socket.send(b"Hi there!").await?;
     ///
-    /// let mut buf = vec![0; 1024];
-    /// let n = socket.recv(&mut buf).await?;
```
Comment (k-nasa) [inline src/net/udp/mod.rs]: Why remove `recv` call?
Is this an example of the `recv` function?

**Comment by spacekookie** [comment]
>-    /// let mut buf = vec![0; 1024];
>-    /// let n = socket.recv(&mut buf).await?;
>
>Why remove `recv` call?
>Is this an example of the `recv` function?

If its not, it should be

File: src/net/udp/mod.rs (hunk 1 above)
Comment (Noah-Kennedy) [reply src/net/udp/mod.rs]: Good catch. It looks like I put the docs in the wrong spot. I'll correct that issue right now

Comment (Noah-Kennedy) [reply src/net/udp/mod.rs]: Fixed it

**Pull request merged**